### PR TITLE
Create sc-multisig-specs.md

### PR DESCRIPTION
**Pull request by luciantodea**

### CODE REVIEW

File: sc-multisig-specs.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Thus, essentially the multisig SC (we will refer to it, from now on, as MSC) enables multiple parties to sign or approve an action that takes place - typically a requirement for certain wallets, accounts, and smart contracts to prevent a rogue or hacked individual from performing detrimental actions.
+
+## Multisig transaction flow
+On-chain multisig wallets are made possible by the fact that smart contracts can call other smart contracts. To execute a multisig transaction the flow would be:
+
+* A proposer or board member proposes an action.
+* The proposed action receives a unique id/hash.
```
Comment (iulianpascalau) [inline sc-multisig-specs.md]: an unique*

File: sc-multisig-specs.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* Pay functions - by default we recommend the MSC to not be set up as payable and any deposit or send transaction of eGLD or ESDT towards the MSC will need to call the desired pay function (if a transaction is not a call to these 2 functions then it is rejected immediately and the value is sent back to original sender): Deposit and/or Send
+
+Any external and internal action will follow these steps and process:
+
+* **Propose action:** this will generate an action id. The action id is unique.
+* **View action:** the board members need to see the action proposed before they approve it.
+* **Sign action:** board members are allowed to sign. We might add an expiration date until board members can sign (until block x…).
+* **Un-sign action:** board members are allowed to un-sign, i.e. to remove their signature from an action. Actions with 0 signatures are cleared from storage. This is to allow mistakes to be cleared.
+* **Perform action (by id/hash)** - can be activated by proposers or board members. It is successful only if enough signatures are present from the board members. Whoever calls “perform action” needs to provide any eGLD required by the target, as well as to pay for gas.
```
Comment (iulianpascalau) [inline sc-multisig-specs.md]: I assume that the value transferred and the transaction fee is taken from the MSC account. If this is correct, we might state this.

File: sc-multisig-specs.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+On-chain multisig wallets are made possible by the fact that smart contracts can call other smart contracts. To execute a multisig transaction the flow would be:
+
+* A proposer or board member proposes an action.
+* The proposed action receives a unique id/hash.
+* N board members are notified (off-chain) to review the action with the specific id/hash.
+* M out of N board members sign and approve the action.
+* Any proposer or board member “performs the action”.
+
+## Design guidelines
+
```
Comment (iulianpascalau) [inline sc-multisig-specs.md:24]: It is not very clear how this is implemented: through a deployable SC or a system (internal) SC. I assume that this is a deployable SC written in Rust and compiled in WASM so we might state that this is a deployable SC.

File: sc-multisig-specs.md (hunk 1 above)
Comment (luciantodea) [reply sc-multisig-specs.md]: ok

**Review by sasurobert — DISMISSED** [review]

File: sc-multisig-specs.md (hunk 2 above)
Comment (luciantodea) [reply sc-multisig-specs.md]: Nope. As mentioned already: Whoever calls “perform action” needs to provide any eGLD required by the target, as well as to pay for gas. If there is a move balance kind of action, who call the action pays the gas and the amount to be moved is taken from MSC balance. But the gas is always taken from the balance on the one who create the "perform action" transaction. Added more details for clarity.

File: sc-multisig-specs.md (hunk 3 above)
Comment (luciantodea) [reply sc-multisig-specs.md:24]: yes, it is a deployable SC written in Rust and compiled in WASM. Added this for clarity.

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**